settings_dialog.js

```diff
@@ -104,11 +146,88 @@ function onSettingsChanged(){
 let browseFilesButton = document.getElementById("browseFilesButton");
 
 browseFilesButton.addEventListener("click", function(event){
-    const target = installPath;
+    const target = settings.installPath;
     console.log("Opening item:", target);
     shell.openItem(target);
 });
 
+function askToMoveFiles(selectedDirectory){
+
+    let files = [];
```

Comment:
This variable should be defined inside the callback. This is filled by that asynchronous operation, it makes no sense that this is outside that scope.